### PR TITLE
Update Idempotency Documentation

### PR DESCRIPTION
**Pull request by Cliftonz**

### CODE REVIEW

File: additional-resources/idempotency.mdx [hunk 1]
```diff
@@ -1,33 +1,119 @@
 ---
 title: 'Idempotent Requests'
-description: 'Learn how to retry API requests without performing the same operation twice'
+description: 'Retry API requests fearlessly, ensuring operation performance just once!'
```
Comment (BiswaViraj) [inline additional-resources/idempotency.mdx:3]:
```suggestion
description: 'Retry API requests fearlessly, ensuring the operation performs just once!'
```
;)

**Review by BiswaViraj — CHANGES_REQUESTED** [review]

File: additional-resources/idempotency.mdx [hunk 2]
```diff
@@ -1,33 +1,119 @@
...
+                to: {
+                subscriberId: '<UNIQUE_SUBSCRIBER_IDENTIFIER>',
+            },
+                payload: {
+                name: "Hello World",
+            },
+                actor: "actorId"
+                tenant: "tenantIdentifier"
+            }
+            );
```
Comment (jainpawan21) [inline additional-resources/idempotency.mdx:82]:
```suggestion
        await novu.trigger("<WORKFLOW_TRIGGER_IDENTIFIER>", {
          to: {
            subscriberId: "<UNIQUE_SUBSCRIBER_IDENTIFIER>",
          },
          payload: {
            name: "Hello World",
          },
          actor: "actorId",
          tenant: "tenantIdentifier",
        });
```

File: additional-resources/idempotency.mdx [hunk 3]
```diff
@@ -1,33 +1,119 @@
...
```
Comment (jainpawan21) [inline additional-resources/idempotency.mdx:111]:
```suggestion
        await novu.trigger("<WORKFLOW_TRIGGER_IDENTIFIER>", {
          to: {
            subscriberId: "<UNIQUE_SUBSCRIBER_IDENTIFIER>",
          },
          payload: {
            name: "Hello World",
          },
          actor: "actorId",
          tenant: "tenantIdentifier",
        });
```

**Review by jainpawan21 — COMMENTED** [review]
JS code formatting issue

**Review by jainpawan21 — APPROVED** [review]

**Pull request merged**